command/registry.go

```diff
@@ -109,40 +114,16 @@ import (
 	"github.com/hashicorp/consul/command/version"
 	"github.com/hashicorp/consul/command/watch"
 
-	"os"
-	"os/signal"
-	"syscall"
-
 	mcli "github.com/mitchellh/cli"
 
 	"github.com/hashicorp/consul/command/cli"
 )
 
-// factory is a function that returns a new instance of a CLI-sub command.
-type factory func(cli.Ui) (cli.Command, error)
-
-// entry is a struct that contains a command's name and a factory for that command.
-type entry struct {
-	name string
-	fn   factory
-}
-
-func createCommands(ui cli.Ui, cmdEntries ...entry) map[string]mcli.CommandFactory {
-	m := make(map[string]mcli.CommandFactory)
-	for _, ent := range cmdEntries {
-		thisFn := ent.fn
-		m[ent.name] = func() (mcli.Command, error) {
-			return thisFn(ui)
-		}
-	}
-	return m
-}
-
-// CommandsFromRegistry returns a realized mapping of available CLI commands in a format that
-// the CLI class can consume. This should be called after all registration is
-// complete.
```

Comment:
This is no longer relevant since we don't register in `init()` any more; we declaratively register commands here